### PR TITLE
Updates 22 persons detection

### PR DESCRIPTION
**Pull request by danleyb2**
@marcbelmont @kal-tek Have below changes so i think the branch is good:

- Added a way to build image with CPU only support. It's smaller than with GPU. I think choosing a different base might break GPU support but i can continue looking into it.
- Added .gitignore for model file
- Updated the README to be more useful and added new info

### CODE REVIEW

**Comment by marcbelmont** [comment]
@kal-tek please test the changes and merge if you have no further comments.

**Review by kal-tek — COMMENTED** [review]
The readme now contains sufficient information.

The docker build using cpu errors out 
image

Using GPU takes a lot of resource. as specified its slow.

**Comment by danleyb2** [comment]
@kal-tek I don't get such an error from my end. It could have been an issue with pypi. Kindly try again:

**Comment by kal-tek** [comment]
> @kal-tek I don't get such an error from my end. It could have been an issue with pypi. Kindly try again: 

image
Still doesnt work. is there anything i missed from the steps?

**Comment by danleyb2** [comment]
@kal-tek I don't think there is a missing step in README. my passing build was a fresh clone on a separate machine. 
Let me try again on a different machine and let you know

**Comment by danleyb2** [comment]
@kal-tek I was able to build on blur demo server too. Have upgraded requirements to `torch==1.7.1+cpu` which appears to exist on your system so kindly pull and try again.

**Comment by kal-tek** [comment]
> @kal-tek I was able to build on blur demo server too. Have upgraded requirements to `torch==1.7.1+cpu` which appears to exist on your system so kindly pull and try again.

I am using Apple M1 pro. The new build also fails for that version as well.

**Review by marcbelmont — APPROVED** [review]
